torch/distributed/_sharding_spec/api.py

```diff
@@ -41,6 +41,34 @@ class ShardingSpec(PlacementSpec):
     pass
 
 
+@dataclass
+class ReshardingSpec(object):
+    """
+    This is a wrapper of PlacementSpec that defines the new placement as being
+    resharded across multiple devices or aggregate into a local tensor.
+
+    The semantics of how a tensor is partitioned is inline with
+    :meth:`torch.chunk`, where ``dim`` in torch.chunk corresponds to the
+    specified ``dim`` and ``chunks`` in torch.chunk is the number of elements
+    in the placement specified.
```

Comment:
This is not true, ReshardingSpec has a ShardingSpec which could be anything. This comment is referring to `ChunkShardingSpec` which might not always be the case.

Reply:
sure, let me change it.